src/resdk/tables/microarray.py

```diff
@@ -0,0 +1,108 @@
+""".. Ignore pydocstyle D400.
+
+========
+MATables
+========
+
+.. autoclass:: MATables
+    :members:
+
+    .. automethod:: __init__
+
+"""
+import os
+from functools import lru_cache
+from typing import Callable, List, Optional
+
+import pandas as pd
+
+from resdk.resources import Collection
+
+from .base import BaseTables
+
+CHUNK_SIZE = 1000
+
+
+class MATables(BaseTables):
+    """A helper class to fetch collection's microarray, qc and meta data.
+
+    This class enables fetching given collection's data and returning it
+    as tables which have samples in rows and microarray / qc / metadata
+    in columns.
+
+    A simple example:
+
+    .. code-block:: python
+
+        # Get Collection object
+        collection = res.collection.get("collection-slug")
+
+        # Fetch collection microarray and metadata
+        tables = MATables(collection)
+        meta = tables.meta
+        exp = tables.exp
+
+    """
+
+    process_type = "data:microarray:normalized"
+    EXP = "exp"
```

Comment:
I would suggest using some other label like `"ma"`. Although the cached file name will not be the same as the hash of data objects is different I would still like it to be different so it's more obvious which cache file contains what.

Reply:
Done.